index.html

```diff
@@ -1,24 +1,137 @@
-<doctype html>
+<!doctype html>
 <html>
 
 <head>  
 <title> </title>
 <style type="text/css">
 	
 </style>
-
+<link type="text/css" rel="stylesheet" href="stylesheet.css" >
 </head>
 
 <body>
+<div>
+  <div id="navbar">
```

Comment:
Instead of having a `id` of `navbar`, why not make it a `nav` element instead. That way the page is more semantic.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav for more details and use case of a `nav` element

Reply:
Thanks for your review. It has been improved on.